### PR TITLE
can auto plot a single number not wrapped as list

### PR DESCRIPTION
**Pull request by galwiner**
resolved issue #156

### CODE REVIEW

File: entropylab/results/dashboard/auto_plot.py [hunk 1]
```diff
@@ -14,6 +15,8 @@ def auto_plot(experiment_id: int, data):
         plot = _auto_plot_from_list(data)
     elif isinstance(data, np.ndarray):
         plot = _auto_plot_from_ndarray(data)
+    elif isinstance(data,int) or isinstance(data,float):
```
Comment (urig) [inline entropylab/results/dashboard/auto_plot.py:18]: Nitpicking: Missing white space after commas (,). :)

If you'd like I can show you how to have `black` fix this for you automatically in PyCharm.

**Review by urig — COMMENTED** [review]
Hi Gal, Can I suggest that you add a unit test for the new use case?

File: entropylab/results/dashboard/auto_plot.py (hunk 1 above)
Comment (galwiner) [reply entropylab/results/dashboard/auto_plot.py:18]: yeah, if we go over it together we can go over both black and unit-test
thanks!

**Review by urig — APPROVED** [review]
LGTM 👍

**Pull request merged**